src/drivers/linux_pwm_out/linux_pwm_out.cpp

```diff
@@ -216,7 +216,13 @@ void task_main(int argc, char *argv[])
 		pwm_out = new NavioSysfsPWMOut(_device, _max_num_outputs);
 	}
 
-	if (pwm_out->init() != 0) {
+	/**
+	 * if using pca9685, _fd should large equal than 0 but in pca9685.h, init method return _fd>=0, it's alway be 1
```

Comment:
please correct this sentence to:
`if using pca9685, _fd should be larger or equal than 0 but in pca9685.h, init method returns _fd>=0, so it would always be equal to 1. The bellow fixes it` - I suppose that's what you meant here